lib/lightning_web/live/credential_live/credential_form_component.ex

```diff
@@ -595,44 +595,14 @@ defmodule LightningWeb.CredentialLive.CredentialFormComponent do
         {name |> Phoenix.HTML.Form.humanize(), name, nil, nil}
       end)
 
-    oauth_clients_from_env =
-      Application.get_env(:lightning, :oauth_clients)
-
     schemas_options
     |> Enum.concat([{"Raw JSON", "raw", nil, nil}])
-    |> handle_oauth_item(
-      {"GoogleSheets", "googlesheets", ~p"/images/oauth-2.png", nil},
-      get_in(oauth_clients_from_env, [:google, :client_id])
-    )
-    |> handle_oauth_item(
-      {
-        "Salesforce",
-        "salesforce_oauth",
-        ~p"/images/oauth-2.png",
-        nil
-      },
-      get_in(oauth_clients_from_env, [:salesforce, :client_id])
-    )
+    # TODO: Gracefully determine which adaptors define custom credential types
+    # and which adaptors use OAuth2 credential types.
+    |> List.delete({"Googlesheets", "googlesheets", nil, nil})
```

Comment:
@taylordowns2000 what is this for ?

Reply:
It removes the Googlesheets option from the list of available credential types.

Reply:
Oh Waouh ! then shouldn't this be handled in the adaptor side ? Like from the configuration schema ?

Reply:
No. Do you remember how `append_if_missing` worked? It was for this reason. I'll call you quick to explain

Reply:
Thanks @taylordowns2000 for the call. All clear for me now.